pennylane/qchem/openfermion_obs.py

```diff
@@ -1013,6 +1014,15 @@ def _(
     load_data=False,
     convert_tol=1e12,
 ):
+
+    if unit == "Angstrom":
+        coordinates = coordinates / bohr_angs
+    elif unit not in ("Angstrom", "Bohr"):
+        raise ValueError(
+            f"The provided unit, '{unit}' is not supported. "
+            f"Please set 'unit' to 'Bohr' or 'Angstrom'."
+        )
+
```

Comment:
```suggestion
    if unit not in ("angstrom", "bohr"):
        raise ValueError(
            f"The provided unit, '{unit}' is not supported. "
            f"Please set 'unit' to 'Bohr' or 'Angstrom'."
        )
        
    if unit == "angstrom":
        coordinates = coordinates / bohr_angs

```

Should make this not case sensitive either.